rotkehlchen/api/rest.py

```diff
@@ -3647,7 +3659,6 @@ def _perform_assets_updates(
 
         if result is None:
             with self.rotkehlchen.data.db.user_write() as cursor:
-                cursor.execute('DELETE from accounts_details;')
```

Comment:
@yabirgb can you please look at this and register it in your mind and the comments in the accompanying commit? https://github.com/rotki/rotki/pull/4982/commits/67ebda2b5b31b7f6d3b566966838a962a784c1a5

This is why I was getting no detected tokens.